integration-test/976-fractional-pois.py

```diff
@@ -47,7 +49,7 @@
 
 assert_has_feature(
     9, 150, 192, 'water',
-    { 'min_zoom': 0, 'id':  10613,
+    { 'min_zoom': 0, 'id': int,
     'source': 'openstreetmapdata.com',
```

Comment:
IDs from the shapefiles aren't necessarily constant - they get automatically assigned either by `shp2pgsql` or Postgres (I'm not sure which). So it's safer to not match on `id` and instead just say it's present and an `int`.

Arguably this is a bug, as it means that prod/dev are different from the test instance. However, when it was written, I couldn't think of an easy way to make sure the IDs are the same.